protocol/messenger_communities.go

```diff
@@ -574,6 +574,21 @@ func (m *Messenger) JoinedCommunities() ([]*communities.Community, error) {
 	return m.communitiesManager.Joined()
 }
 
+func (m *Messenger) CommunityUpdateLastOpenedAt(communityID string) (*MessengerResponse, error) {
+	community, err := m.communitiesManager.GetByIDString(communityID)
+	if err != nil {
+		return nil, err
+	}
+	err = m.communitiesManager.UpdateLastOpenedAt(community.ID())
+	if err != nil {
+		return nil, err
+	}
+	community.UpdateLastOpenedAt()
+	response := &MessengerResponse{}
+	response.AddCommunity(community)
+	return response, nil
+}
+
 func (m *Messenger) SpectatedCommunities() ([]*communities.Community, error) {
 	return m.communitiesManager.Spectated()
 }
```

Comment:
Would it be possible to change this function so that it matches `SpectatedCommunities()`? The reason is any changes to the community should be handled by the communitiesManager, and not the Messenger

Reply:
Done